WordPress/Classes/ViewRelated/New Group/SiteSegments/SiteSegmentsCell.xib

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I recognize that we use XIBs & storyboards elsewhere in the app? I like these tools, but I find the XML to be inscrutable, especially during PR reviews.

How would you feel about doing all of our layout programmatically?